lib/persisted-model.js

```diff
@@ -1489,7 +1489,6 @@ module.exports = function(registry) {
 
   function getIdFromWhereByModelId(Model, where) {
     var whereKeys = Object.keys(where);
-    if (whereKeys.length != 1) return undefined;
 
     var idName = Model.getIdName();
     if (whereKeys[0] !== idName) return undefined;
```

Comment:
This assumes the id is the first key, which is not always the case.

Consider the following: `where: { name: 'Miroslav', id: 123 }`. IIUC, such filter would not be recognised as updating a single property only.